app/views/user_mailer/send_email.html.erb

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <h1>Sorry to inform you, <%= @recipient.name %></h1>
-    <p><%= @user.name %> has died :( </p>
+    <p><%= @user.name %> has passed away. <%= @user.name %> cared for you very much, and wanted you to be notified of this unfortunate circumstance.</p>
     <p> Here is a link to <%= @user.name %>'s   <a href =<%= @user_url %>> Memorial Page </a> </p>
```

Comment:
Is it possible to link to the executors name and contact info in case the recipient has any questions?

Reply:
I think so, let me check on that